coda-prototype/.gitignore

```diff
@@ -2,3 +2,4 @@
 node_modules/
 coverage/
 coda*.zip
+dist/
```

Comment:
Convention is to call this 'out' or 'build'

Reply:
Even for TypeScript? Microsoft's sample projects use dist:
https://github.com/Microsoft/TypeScript-Node-Starter